docs/earthly-command/earthly-command.md

```diff
@@ -282,6 +282,69 @@ Instructs earthly to issue a "prune all" command to the buildkit daemon.
 
 Restarts the buildkit daemon and completely resets the cache directory.
 
+## earthly config
+
+#### Synopsis
+
+```
+earthly [options] config [key] [value]
+```
+
+#### Description
+
+Enables modifying, adding, and updating values in `~/.earthly/config.yml`. It does its best to preserve formatting and existing comments. You can also set complex values using YAML literals.
```

Comment:
I wonder if we can make this more concise like this?
```suggestion
Manipulates values in `~/.earthly/config.yml`. The `value` parameter can be a string or YAML literals.
```

Reply:
Its not just strings, its literally only YAML literals. Which include bare numbers and boolean values too. I can make the first word manipulates.

Reply:
I shortened it some?